### PR TITLE
Dp tackling old news

### PR DESCRIPTION
**Pull request by duncdrum**
going through some old issues, but @wolfgangmm might want to take a look at the updated repo.xml documentation

### CODE REVIEW

File: src/main/xar-resources/data/troubleshooting.xml [hunk 1]
```diff
@@ -27,29 +27,29 @@
...
-06 Oct 2012 16:56:32,824 [main] INFO  (JettyStart.java [run]:127) - [eXist Version : 2.1-dev] 
-06 Oct 2012 16:56:32,824 [main] INFO  (JettyStart.java [run]:129) - [eXist Build : 20121006] 
-06 Oct 2012 16:56:32,825 [main] INFO  (JettyStart.java [run]:131) - [eXist Home : unknown] 
-06 Oct 2012 16:56:32,825 [main] INFO  (JettyStart.java [run]:133) - [SVN Revision : 17031] 
-06 Oct 2012 16:56:32,825 [main] INFO  (JettyStart.java [run]:141) - [Operating System : Mac OS X 10.8.2 x86_64] 
-06 Oct 2012 16:56:32,825 [main] INFO  (JettyStart.java [run]:144) - [jetty.home : /Applications/eXist/tools/jetty] 
-06 Oct 2012 16:56:32,825 [main] INFO  (JettyStart.java [run]:146) - [log4j.configuration : file:/Applications/eXist/log4j.xml] 
-06 Oct 2012 16:56:33,698 [main] INFO  (JettyStart.java [lifeCycleStarting]:387) - Jetty server starting... 
-06 Oct 2012 16:56:33,699 [main] INFO  (Server.java [doStart]:253) - jetty-8.1.3.v20120416 
+06 Oct 2012 16:56:32,797 [main] INFO  (JettyStart.java [run]:116) - Configuring eXist from /Applications/eXist/conf.xml
```
Comment (dizzzz) [inline src/main/xar-resources/data/troubleshooting.xml]: we should update this transcript

**Comment by dizzzz** [comment]
what is "DP" ?

**Comment by duncdrum** [comment]
my initials

File: src/main/xar-resources/data/troubleshooting.xml (hunk 1 above)
Comment (duncdrum) [reply src/main/xar-resources/data/troubleshooting.xml]: you should be able to modify my PR, no?

**Pull request merged**